src/Workspaces/Remote/Core/ExternalAccess/UnitTesting/Api/UnitTestingAssetStorageWrapper.cs

```diff
@@ -7,6 +7,20 @@
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
+    internal readonly struct UnitTestingAssetStorageWrapper
+    {
+        public static UnitTestingAssetStorageWrapper Instance { get; } = new UnitTestingAssetStorageWrapper(AssetStorage.Default);
+
+        internal AssetStorage UnderlyingObject { get; }
+
+        internal UnitTestingAssetStorageWrapper(AssetStorage underlyingObject)
+            => UnderlyingObject = underlyingObject ?? throw new ArgumentNullException(nameof(underlyingObject));
+
+        public void UpdateLastActivityTime()
+            => UnderlyingObject.UpdateLastActivityTime();
+    }
```

Comment:
Not needed anymore, right?

Reply:
Yes, and I think I have removed this one in this commit https://github.com/dotnet/roslyn/pull/43856/commits/b8d8811383148cee5434c662459323c3d811426e
You might not get it refreshed?